### DISCUSSION TITLE
How can I set the size of the sheet squares to be smaller squares ?

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#4617] [Q&A]
When I use the graphic tablet with a zoom size of 170 approximately when I write with the pencil things, like measures, angles, etc. of small size, it is difficult to be very precise, etc of small size it is difficult for me to be very precise that is why I use the Zoom to 250 or 300 that allows me to have a better control of the writing but the bad thing is that when making zoom of that size the grids look very big when I make geometric figures it occupies me a lot of space since the figures look better when they fit in the grids that is why I would like to know if there is a way to reduce the size of the grids.

so that you understand me I have here an image of my xournal++ and the whiteboard grids with a zoom of 250 approximately.
![image](https://user-images.githubusercontent.com/28522011/214192323-d7279067-5eef-458a-aa49-f64aa76ef4a1.png)

**Comment by rolandlo** [comment]
You can add a custom page template (to `pagetemplates.ini`). See https://github.com/xournalpp/xournalpp/issues/2137 and in particular https://github.com/xournalpp/xournalpp/issues/2137#issuecomment-799956788
You can also adjust the grid size for snapping in the drawing area tab of menu `Edit > Preferences`.
